src/test/regress/input/multi_alter_table_statements.source

```diff
@@ -356,10 +355,30 @@ FROM
 ORDER BY attnum;
 \c - - - :master_port
 
+-- verify that we can rename distributed tables
+SHOW citus.enable_ddl_propagation;
+ALTER TABLE lineitem_alter RENAME TO lineitem_renamed;
+
+-- verify rename is performed
+SELECT relname FROM pg_class WHERE relname = 'lineitem_renamed';
+
+-- show rename worked on one worker, too
+\c - - - :worker_1_port
+SELECT relname FROM pg_class WHERE relname LIKE 'lineitem_renamed%';
+\c - - - :master_port
+
+-- revert it to original name
+ALTER TABLE lineitem_renamed RENAME TO lineitem_alter;
+
+-- show rename worked on one worker, too
+\c - - - :worker_1_port
+SELECT relname FROM pg_class WHERE relname LIKE 'lineitem_alter%';
```

Comment:
It might make sense to add `ORDER BY relname` to prevent the output from changing in different environments (e.g. Travis). I think we need to change existing cases that omit the ORDER BY, but it's a good practice to follow going forward.

Reply:
And Travis is failing right on that here. I'm only going to edit new tests in this PR, though.